protocols/raft/src/main/java/io/atomix/protocols/raft/impl/RaftServiceManager.java

```diff
@@ -470,6 +473,8 @@ private void applyIndex(long index) {
             future.complete((T) applyInitialize(entry.cast()));
           } else if (entry.type() == ConfigurationEntry.class) {
             future.complete((T) applyConfiguration(entry.cast()));
+          } else if (entry.type() == ZeebeEntry.class) {
+            future.complete(null);
```

Comment:
Did we already write this entry?

Reply:
Was just wondering why it is in this pr

Reply:
In the spike I was doing it with our own `RaftStateMachine` implementation, and here it would fail the operation (which in Atomix essentially does nothing anyway :man_shrugging:). Since we're committed to our fork, I'd like to avoid having to use our own `RaftStateMachine` impl. in the future, so I added it here. See #49